swerve100/src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java

```diff
@@ -197,7 +197,7 @@ public SwerveDriveSubsystem(DriverStation.Alliance alliance, double currentLimit
                 kMaxSpeedMetersPerSecond = 5;
                 kMaxAccelerationMetersPerSecondSquared = 10;
                 kMaxAngularSpeedRadiansPerSecond = 5;
-                kMaxAngularSpeedRadiansPerSecondSquared = 4.5;
+                kMaxAngularSpeedRadiansPerSecondSquared = 5;
```

Comment:
5 rad/s/s seems low to me if max speed is 5 rad; it seems like every motion will be accel limited.  is that what you want?